ovsnl/client.go

```diff
@@ -125,3 +125,20 @@ func (c *Client) initFamily(f genetlink.Family) error {
 
 	return fmt.Errorf("unrecognized OVS generic netlink family: %q", f.Name)
 }
+
+// headerBytes converts an ovsh.Header into a byte slice.
+func headerBytes(h ovsh.Header) []byte {
+	b := *(*[sizeofHeader]byte)(unsafe.Pointer(&h))
+	return b[:]
```

Comment:
Endianness doesn't matter here since these are only for use with netlink which uses host order, is that correct?

Reply:
Yep, exactly.